File: ejercicio_6/CelsiusAFahrenheit.java
import java.util.Scanner;

public class CelsiusAFahrenheit {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        System.out.print("Digite la temperatura en grados Celsius (C°): ");
        double celsius = scanner.nextDouble();
        
        double fahrenheit = (9.0 / 5.0) * celsius + 32;
        
        System.out.println("La cantidad en grados Fahrenheit (F) es: " + fahrenheit);
        
        scanner.close();
    }
}

File: ejercicio_6/celsius_a_fahrenheit.py
def main():
    try:
        celsius = float(input("Digite la temperatura en grados Celsius (C°): "))
        
        fahrenheit = (9 / 5) * celsius + 32
        
        print(f"La cantidad en grados Fahrenheit (F) es: {fahrenheit}")
    except ValueError:
        print("Número invalido")

if __name__ == '__main__':
    main()
